helpers.py

```diff
@@ -1,5 +1,8 @@
 from math import log, sin
 
 
-def calc(x):
-    return str(log(abs(12 * sin(int(x)))))
+def solve_captcha(number: int):
+    """this function helps to calculate math expression,
```

Comment:
Хорошо! Ещё пара мыслей есть вдобавок.


Если убрать первые несколько слов, смысл потеряется?

```python
"""Calculate math expression, which value will be used to solve captcha."""
```

Также из источника, указанного в #3:
> PEP 257 also says to start your docstrings with a capital letter, use the imperative tense (meaning "return" instead of "returns"), end with period, and a whole bunch of other stuff.

---
И да, "Witch | wɪtʃ |" — "ведьма".

"Which | wɪtʃ |" — "который".

🤷‍♂️